### PR TITLE
Add GAME and CRW

### PR DESCRIPTION
**Pull request by mikeyb**

### CODE REVIEW

File: keyconv.c [hunk 1]
```diff
@@ -854,6 +855,13 @@ main(int argc, char **argv)
 					privtype_opt = 166;
 					break;
 			}			
+			if (strcmp(optarg, "CRW")== 0) {
```
Comment (exploitagency) [inline keyconv.c]: else
if (...

File: oclvanitygen.c [hunk 2]
```diff
@@ -927,6 +928,13 @@ main(int argc, char **argv)
 					privtype = 166;
 					break;
 			}			
+			if (strcmp(optarg, "CRW")== 0) {
```
Comment (exploitagency) [inline oclvanitygen.c]: else
if (...

File: vanitygen.c [hunk 3]
```diff
@@ -1179,6 +1179,13 @@ main(int argc, char **argv)
 					privtype = 166;
 					break;
 			}			
+			if (strcmp(optarg, "CRW")== 0) {
```
Comment (exploitagency) [inline vanitygen.c]: else
if (...

Comment (exploitagency) [inline vanitygen.c]: also missing help dialog

**Comment by mikeyb** [comment]
My bad.  Thanks for grabbing that, missed your reply.  I will make sure to pay closer attention on future adds

**Comment by exploitagency** [comment]
Its really a non issue, next time you send a PR just give me rights to change things, theres some check box when sending a PR "Allow edits from maintainers", and I will fix anything or do formatting as I see fit and merge it upon review.  I just went ahead and merged it then manually made the changes and issued a new binary release, the PinkCoin team wanted their coin on here.

Thanks for the contribution by the way!  It is hard to pick and choose what to add when there are so many altcoins so the community is helpful.

https://github.com/exploitagency/vanitygen-plus/releases

PS: I didn't test your values, but I trust your judgment as a dev.

**Pull request merged**